packages/patterns/src/patterns/patternMatchers.js

```diff
@@ -934,7 +958,7 @@ const makePatternKit = () => {
   const matchRemotableHelper = Far('match:remotable helper', {
     checkMatches: (specimen, remotableDesc, check) => {
       // Unfortunate duplication of checkKind logic, but no better choices.
```

Comment:
Does this comment still apply?

Reply:
The comment is no longer informative. Gone.